contents/docs/experiments/funnels-statistics.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Statistical methodology for funnel experiments
+---
+
+Funnel experiments use Bayesian statistics with a beta model to evaluate the **win probabilities** and **credible intervals** for an experiment. [Read the statistics primer for an overview](/docs/experiments/statistics-primer) if you haven't already.
+
+## What the heck is a Beta model?
+
+Imagine you run a pizza shop and want to know if customers say "yes" to adding pineapple. Some customers will say yes, others will say no.
+
+The **beta distribution** is a statistical model that's great for analyzing proportions or probabilities. It helps us understand:
+1. The true probability of customers saying yes to adding pineapple.
+2. How certain we are about that probability.
+
+For example, if:
+- Only 2 out of 4 customers (50%) say yes, the Beta distribution will be wide, indicating high uncertainty.
+- 150 out of 300 customers (50%) say yes, the Beta distribution will be narrow, showing we're more confident about that 50% rate.
+
+Within PostHog, our Beta model uses a minimally informative prior of `ALPHA_PRIOR = 1` and `BETA_PRIOR = 1`.
+
+## Win probabilities
+
+The **win probability** tells you how likely it is that a given variant has the highest conversion rate compared to all other variants in the experiment. It helps you determine whether the experiment shows a **statistically significant** real effect vs. simply random chance.
+
+Let's say you're testing a new signup flow and have these results:
```

Comment:
Could we use the same pineapple on pizza example? If it doesn't work here, maybe we should use one that works for both.

Reply:
Maybe pizza website? Experiment to upsell pineapple?

Reply:
Yeah, good catch. Updated the examples with bb2d23c12d65d9b71b94df50b0126247e435b7e4 and b20beae809fe76f85c40a03acfdb7ba0b676edf5